README.md

```diff
@@ -1,66 +1,77 @@
-	- [Esri.ArcGISRuntime.Toolkit.Xamarin.Forms](https://www.nuget.org/packages/Esri.ArcGISRuntime.Toolkit.Xamarin.Forms)
-	- [Esri.ArcGISRuntime.ARToolkit](https://www.nuget.org/packages/Esri.ArcGISRuntime.ARToolkit)
-	- [Esri.ArcGISRuntime.ARToolkit.Forms](https://www.nuget.org/packages/Esri.ArcGISRuntime.ARToolkit.Forms)
-2. **[Build from source](https://esri.github.io/arcgis-toolkit-dotnet/buildingtoolkit.html)** - do this if you want to customize toolkit
-   
 ## Features
 
-> See [List of controls](https://esri.github.io/arcgis-toolkit-dotnet/controls.html) for a full list of controls with screenshots
-
-- **[ARSceneView](https://esri.github.io/arcgis-toolkit-dotnet/ar.html)**: Part of the AR Toolkit, enables integration of GIS content and ARKit/ARCore.
-- **[BasemapGallery](https://esri.github.io/arcgis-toolkit-dotnet/basemap-gallery.html)**: Shows basemaps, either from a Portal or a custom collection; applies the selected basemap to the connected GeoModel.
-- **[Bookmarks](https://esri.github.io/arcgis-toolkit-dotnet/bookmarks-view.html)**: Shows bookmarks, from a map, scene, or a list; navigates the associated MapView/SceneView when a bookmark is selected.
-- **Compass**: Shows a compass direction when the map is rotated. Auto-hides when the map points north up.
-- **FeatureDataField**: Displays and optionally allows editing of a single field attribute of a feature.
-- **Legend**: Displays a legend for a single layer in your map (and optionally for its sub layers).
-- **MeasureToolbar**: Allows measurement of distances and areas on the map view.
-- [**OverviewMap**](https://esri.github.io/arcgis-toolkit-dotnet/overview-map.html): Displays an interactive inset map for a map or scene.
-- **PopupViewer**: Display details and media, edit attributes, geometry and related records, and manage the attachments of features and graphics (popups are defined in the popup property of features and graphics).
-- **ScaleLine**: Displays current scale reference.
-- **[SearchView](https://esri.github.io/arcgis-toolkit-dotnet/search-view.html)**: Enables searching using one or more locators, with support for suggestions, automatic zooming, and custom search sources.
-- **SymbolDisplay**: Renders a symbol in a control.
-- **TimeSlider**: Allows interactively defining a temporal range (i.e. time extent) and animating time moving forward or backward.  Can be used to manipulate the time extent in a MapView or SceneView.
-
-### Features in Preview
+|Component |UWP  |WPF  |Xamarin.Android  |Xamarin.iOS  |Xamarin.Forms |
+|---|---|---|---|---|---|
+|[ARSceneView](https://esri.github.io/arcgis-toolkit-dotnet/ar.html)   | N/A | N/A | ✔ | ✔ | ✔ |
+|[BasemapGallery](https://esri.github.io/arcgis-toolkit-dotnet/basemap-gallery.html) | ✔ | ✔ | ❌ | ❌ | ✔ |
+|[BookmarksView](https://esri.github.io/arcgis-toolkit-dotnet/bookmarks-view.html)   | ✔ | ✔ | ✔ | ✔ | ✔ |
+|Compass   | ✔ | ✔ | ✔ | ✔ | ✔ |
```

Comment:
Can we add doc links to all components in the table?

Reply:
I would like to, but unfortunately we don't have guide doc for most components. I've been back filling doc as I have time.